bundles/org.openhab.core/src/main/java/org/openhab/core/util/ColorUtil.java

```diff
@@ -63,7 +63,7 @@ public static int[] hsbToRgb(HSBType hsb) {
      * Transform <a href="https://en.wikipedia.org/wiki/HSL_and_HSV">HSV</a> based {@link HSBType} to
      * <a href="https://en.wikipedia.org/wiki/SRGB">sRGB</a>.
      *
-     * This function not round the components to integer values. Please consider using {@link #hsbToRgb(HSBType)}
+     * This function does not round the components to integer values. Please consider using {@link #hsbToRgb(HSBType)}
      * whenever integer values are required.
```

Comment:
Maybe this should be rewritten to something like `This method does not round the components. For conversion to integer values in the range of 0-255 use ...`,

Reply:
Ok.